spreadsheet/formula/functions_test.go

```diff
@@ -0,0 +1,328 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+//
+// Use of this source code is governed by the terms of the Affero GNU General
+// Public License version 3.0 as published by the Free Software Foundation and
+// appearing in the file LICENSE included in the packaging of this file. A
+// commercial license can be purchased by contacting sales@baliance.com.
+
+package formula_test
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/unidoc/unioffice/measurement"
+	"github.com/unidoc/unioffice/schema/soo/sml"
+	"github.com/unidoc/unioffice/spreadsheet"
+	"github.com/unidoc/unioffice/spreadsheet/formula"
+)
+
+type testStruct struct {
+	Inp string
+	Exp string
+}
+
+func runTests(t *testing.T, ctx formula.Context, td []testStruct) {
+	ev := formula.NewEvaluator()
+	for _, tc := range td {
+		t.Run(tc.Inp, func(t *testing.T) {
+			p := formula.Parse(strings.NewReader(tc.Inp))
+			if p == nil {
```

Comment:
Might be nice to use testify require.  require.NotNil(t, p)
etc

Reply:
golang standard testing package doesn't contain assert or require methods, do you consider to use github.com/stretchr/testify/require ?

Reply:
Yes I was referencing that one.

Reply:
Doesn't matter much in this specific case here, but can be nice in longer test cases with many
comparisons etc.